ntbs-service/DataMigration/CaseManagerImportService.cs

```diff
@@ -47,6 +47,7 @@ public class CaseManagerImportService : ICaseManagerImportService
             ntbsCaseManager.GivenName = legacyCaseManager.GivenName;
             ntbsCaseManager.FamilyName = legacyCaseManager.FamilyName;
             ntbsCaseManager.DisplayName = $"{ntbsCaseManager.GivenName} {ntbsCaseManager.FamilyName}";
+            ntbsCaseManager.IsCaseManager = true;
```

Comment:
Why did this need to change?

Reply:
In the ACs it says that if a notification is imported and the user exists in NTBS this field should be set to true. It does no harm to also set this on legacy users, they are going to be inactive anyway so they won't show in dropdowns/service directory. As we are assigning a notification to this user I think it makes sense to always set it to true

Reply:
Hmm... my gut feeling is that it's a bit surprising to have this field set to true for legacy case managers. Having said that, I can't really think of a good reason why, so happy to leave this as it is.